### DISCUSSION TITLE
Problems installing with npm

### DISCUSSION THREAD

**Discussion by Assblastois** [vueform/vueform#233] [Questions] [answered]
I encounter this error when trying to install vueform via: npm create vueform@latest

Creating project 'vueform-project' using Vite...    
Failed to create project with Vite: spawn npm ENOENT

Does anyone know why I have this problem? I could install VueJS without any problems. 
Any help is welcome, thanks!

**Comment by Leremust** [comment]
I have the same issue since yesterday, any help please

**Comment by adamberecz** [comment, marked as answer]
Thanks for reporting the issue. A fix has been [released](https://github.com/vueform/create-vueform/releases) - please update `create-vueform` to `1.0.12` (or run `npm create vueform@latest`) and try again.

**Reply by Leremust** [reply]
This fixed the issue. Thanks
